### PR TITLE
Update to styled components alpha and fix search

### PR DESCRIPTION
**Pull request by taysea**
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?

Updates to use styled components 6 alpha release. Also fixes search bug introduced based on changes to sentence casing page titles.

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

**Review by britt6612 — APPROVED** [review]
good remembering to fix links

File: package.json [hunk 1]
```diff
@@ -30,12 +30,12 @@
...
   ],
   "dependencies": {
-    "grommet": "https://github.com/grommet/grommet/tarball/stable",
+    "grommet": "^2.35.0-alpha.1",
     "grommet-icons": "https://github.com/grommet/grommet-icons/tarball/stable",
     "grommet-theme-hpe": "https://github.com/grommet/grommet-theme-hpe/tarball/stable",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "styled-components": "^5.3.6"
+    "styled-components": "^5.1.0"
```
Comment (jcfilben) [inline package.json:38]: Do we want to try and use styled-components v6 on the site in addition to using the grommet alpha version?

Comment (taysea) [reply package.json:38]: Whoops..  this was meant to be 6.1.0. Will update!

Comment (taysea) [reply package.json]: Fixed.

**Review by jcfilben — APPROVED** [review]

**Pull request merged**